### ISSUE TITLE
Add section on tools on landing page

### ISSUE THREAD

**Issue by uekerman** [precice/precice.github.io#268]
Two important tools around preCICE have reached a certain level of maturity such that we should advertise them more prominently. They might attract new user communities:
- Micro Manager
- FMI-preCICE Runner

We could add a dedicated section on such tools on the landing page, maybe between "adapters" and "what our community is saying". With short explanations on what users can do with these tools:
- "Perform many-to-one macro-micro simulations with the preCICE Micro Manager"
- "Couple FMUs to PDEs with the FMI-preCICE Runner"

Each tool could become a card with a visual (with a similar style as the features on the landing page), such that we could also extend this list in the future.

Links should go to dedicated pages under `docs/tooling` (WIP for both tools).

I am not sure whether to add more of our tools here (config visualizer, ASTE, ...). These are hard to understand without knowing preCICE already.

**Comment by MakisH** [comment]
> Each tool could become a card with a visual (with a similar style as the features on the landing page)

Actually, why not simply extend the "features" section? Considering these tools as part of the preCICE ecosystem, these are features of what the user sees as "preCICE".

As we need more and more sections on the landing page, I am trying to think of ways to reduce their number, to not make the page too long.

**Comment by uekerman** [comment]
Could be an alternative, yes.

**Comment by MakisH** [comment]
We could have a section "Versatile coupling" between "prepared ..." and "ready-to-use"

- Surface and volume coupling (needs documentation entrypoint #473)
- Coupling to FMUs
- Coupling to particles
- Many-to-one coupling
- Mixed dimensions

The "Learn more" could link to a new section in https://precice.org/docs.html explaining how to approach the documentation depending on the coupling type you need.

I could then design fitting pictures.

What do you think?

**Comment by uekerman** [comment]
> We could have a section "Versatile coupling" between "prepared ..." and "ready-to-use"

Sounds good.

And then a card for each of your items? Sounds good to me.

> The "Learn more" could link to a new section in https://precice.org/docs.html explaining how to approach the documentation depending on the coupling type you need.

This point I don't understand yet. It would be one "learn more" for each card, right? For some of your items, we would already have good landing pages.

**Comment by MakisH** [comment]
> > The "Learn more" could link to a new section in https://precice.org/docs.html explaining how to approach the documentation depending on the coupling type you need.
> 
> This point I don't understand yet. It would be one "learn more" for each card, right? For some of your items, we would already have good landing pages.

Yes, one for each card. Leftover from my first draft, where I was thinking of one feature card for all.
